packages/webapp/components/layouts/AccountLayout/Security/index.tsx

```diff
@@ -0,0 +1,142 @@
+}: AccountSecurityDefaultProps): ReactElement {
+  const { user } = useContext(AuthContext);
+  const [email, setEmail] = useState<string>();
+  const [resetPasswordSent, setResetPasswordSent] = useState(false);
+
+  const emailAction = isEmailSent ? (
+    <EmailSentSection />
+  ) : (
+    <Button
+      className="mt-6 w-fit btn-secondary"
+      onClick={() => onSwitchDisplay(Display.ChangeEmail)}
+    >
+      Change email
+    </Button>
+  );
+
+  const onDelete = () => {};
+
+  return (
+    <AccountPageContainer title="Security">
+      <AccountContentSection
+        headingClassName="mt-0"
+        title="Account email"
+        description="The email address associated with your daily.dev account"
+      >
+        <AccountTextField
+          fieldType="tertiary"
+          value={email ?? user.email}
+          onChange={(e) => setEmail(e.currentTarget.value)}
+          label="Email"
+          inputId="email"
+          leftIcon={<MailIcon />}
+          rightIcon={<LockIcon className="text-theme-label-secondary" />}
+          readOnly
+        />
+        {emailAction}
+      </AccountContentSection>
+      <AccountLoginSection
+        title="Add login account"
+        description="Add more accounts to ensure you never lose access to your daily.dev
+        profile and to make login quick and easy cross device"
+        providers={providers.filter(
+          ({ provider }) => !user.providers.includes(provider.toLowerCase()),
+        )}
+        action={
+          !user.password && (
+            <ProviderButton
+              label="Connect with"
+              provider="Email"
+              icon={<MailIcon secondary />}
+              onClick={() => onSwitchDisplay(Display.ConnectEmail)}
+              style={socialProvider.gitHub.style}
+            />
+          )
+        }
+      />
+      <AccountLoginSection
+        title="Connected accounts"
+        description="Remove the connection between daily.dev and authorized login providers."
+        providers={providers.filter(({ provider }) =>
+          user.providers.includes(provider.toLowerCase()),
+        )}
+        action={
+          user.password && (
+            <ProviderButton
+              label="Remove"
+              provider="Email"
+              icon={<MailIcon secondary />}
+              onClick={() => onSwitchDisplay(Display.ConnectEmail)}
+              className="bg-theme-bg-tertiary hover:bg-theme-status-error"
+            />
+          )
+        }
+      />
+      {/* {user.password && ( // temporary enabling */}
+      <AccountContentSection
+        title="Account Password"
+        description="Change your account password"
+      >
+        <Button
+          className="mt-6 w-fit btn-secondary"
+          onClick={() => setResetPasswordSent(true)}
+        >
+          Reset password
+        </Button>
+      </AccountContentSection>
+      {/* )} */}
+      {resetPasswordSent && (
+        <OverlayContainer className="mt-6">
```

Comment:
This could be seperate/modal?

Reply:
![image](https://user-images.githubusercontent.com/13744167/183613502-1c5ff766-ae5e-4202-b422-92df2c181e9f.png)
This overlay portion is the one below.